### ISSUE TITLE
trakt_lookup didn't work correctly sometimes

### ISSUE THREAD

**Issue by ProteinPig** [Flexget/Flexget#2805]
I use trakt_lookup to filter movies, and expect movies marked as watched or collected don't add to a movie_list "wanted movies". 

But, I still got movies that I have already marked as watched or collected. 

The task runs every 15 mins, is that a cache problem or just a bug? 

Here is the config 
trakt_lookup: 
  username: name 
  account: name 
if:
  - trakt_watched: reject 
  - trakt_collected: reject

**Comment by soloam** [comment]
Yes , if you run your task every 15min you will bump with cache sync problems for sure

Depending on trakt is never a good idea... It's a very unstable service. Recommendation, try to run a task that checks if the movie is  marked as watched in trakt and if yes you can remove it from the "wanted movies" list. This way even if you bump into old data from cache or trakt sync problems, eventually in one of the next runs it will remove it from the list.
